README.wine.md

```diff
@@ -1,40 +1,98 @@
+64-bit wine is under somewhat active development so a recent installation is
+recommended. Fedora keeps it up-to-date and ships version 3.15 of both
+32-bit and 64-bit as of the time of writing. This also allow `WOW64` i.e.
+runnning both 32-bit and 64-bit window binaries in the same setting.
+
 For the development of HarfBuzz, the Microsoft shaping technology, Uniscribe,
 as a widely used and tested shaper is used as more-or-less OpenType reference
 implemenetation and that specially is important where OpenType specification
 is or wasn't that clear. For having access to Uniscribe on Linux/macOS these
 steps are recommended:
 
-1. Install Wine from your favorite package manager.
+1. Install Wine and mingw32/mingw64 from your package manager. This likely
+   means, at the very least, running this:
+
+```
+sudo dnf install -y wine.i686 wine.x86_64 mingw32-gcc-c++ mingw64-gcc-c++
```

Comment:
Can you put its Debian (Ubuntu to be precise equivalent of it) also? Or make the guide that general to cover that also

Reply:
I'll need to look up a bit on this.